frontend/src/Components/SavedNavigationButton/SavedNavigationButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const SavedNavigationButton = ({ navigation, navigationPage, disabled }) => {
+  return disabled ? (
+    <span className='page-item disabled'>
+      <span className='page-link' aria-hidden='true'>
+        {navigation}
+      </span>
+    </span>
+  ) : (
+    <Link
+      className='page-item'
+      to={{

```

Comment:
I don't know why eslint is forcing this on a separate line, this is pretty unclear.